common/changes/@aws/swb-reference/janeyu-userCheck_2023-06-12-22-00.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Remove this file and label this PR as `skip-verify-changefiles` since we don't want any new change files in `release/v2.0.0` now that we're preparing this to be merged into main.